src/utilities/dockerUtils.ts

```diff
@@ -23,7 +23,12 @@ export async function getDeploymentConfig(): Promise<DeploymentConfig> {
       )
    }
 
-   const imageNames = core.getInput('images').split('\n') || []
+   let imageNames = core.getInput('images').split('\n') || []
+   imageNames.forEach((element) => {
+      if (element == ' ' || '') {
```

Comment:
we should be checking if the element is whitespace. This is checking if it's ' ' and if it's not it defaults to '' which is false. This will only enter the if statement if the element is ' '. Instead we want to be removing any elements in the imageNames array that is just whitespace

Reply:
You can test this out using unit tests or any Node shell. I like to use just the `node` command on the cli to experiment what happens if I'm ever unsure of something's behavior